app/vmagent/README.md

```diff
@@ -1029,6 +1029,124 @@ If you have suggestions for improvements or have found a bug - please open an is
 
 See also [troubleshooting docs](https://docs.victoriametrics.com/Troubleshooting.html).
 
+## Google PubSub integration
+[Enterprise version](https://docs.victoriametrics.com/enterprise.html) of `vmagent` can read and write metrics from / to google [PubSub](https://cloud.google.com/pubsub):
+
+
+### Reading metrics from PubSub
+
+[Enterprise version](https://docs.victoriametrics.com/enterprise.html) of `vmagent` can read metrics in various formats from Pub/Sub messages.
+These formats can be configured with `-gcp.pubsub.subscribe.defaultMessageFormat` or `-gcp.pubsub.subscribe.topicSubscription.messageFormat` command-line options. The following formats are supported:
```

Comment:
`-gcp.pubsub.subscribe.defaultMessageFormat` and `-gcp.pubsub.subscribe.topicSubscription.messageFormat` allow to configure the message format 
We support the following options: